docs/Developers/Newcomers_Guide.rst

```diff
@@ -87,7 +87,23 @@ Step 1. Meet the Community!
 
 To get started, the first step is to meet the community. We use gitter to
 communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
+Gitter is instant messaging service used by developers and users of GitHub.
+Gitter uses chatrooms, where developers can join in and can talk about particular topic.
+Coala has several chatrooms. Here are a few of them. 
+
+* `coala <https://gitter.im/coala/coala>`_
+* `gsoc <https://gitter.im/coala/coala/gsoc>`_
+* `cobot-test <https://gitter.im/coala/cobot-test>`_
+* `coala-bears <https://gitter.im/coala/coala-bears>`_
+* `workshop <https://gitter.im/coala/coala/workshops>`_
+* `freelancers <https://gitter.im/coala/freelancers>`_
+
+But before joining the community, here are few things that you should keep in mind.
+
+* Don't @-mention or private message people, unless its utterly important
```

Comment:
It's important to explain why that is: @ mentions generate notifications on the various gitter clients the user may be signed into, you might even wake someone on the other side of the world up. Also it discourages other people to answer the question, so you might wait longer for an answer.

Please add something like this here.